modules/ROOT/pages/security.adoc

```diff
@@ -0,0 +1,252 @@
+
+include::partial$_std-hdr-sgw.adoc[]
+:topic-group: Security
+:param-related:  {authentication-users--xref}  |  {authentication-certs--xref}
+:param-abstract!:
+include::partial$block-abstract.adoc[]
+
+:svc-config-json: https://github.com/couchbase/sync_gateway/blob/master/examples/serviceconfig.json[blob serviceconfig.json]
+
+
+== Overview
+
+From {sgw-te} 3.0, TLS is enabled by default on {sgw}.
+TLS v1.3 is supported as the default protocol.
+
+This approach means that all endpoints communicating with {sgw} must be encrypted by default, including:
+
+* Client side network traffic, for example through Couchbase Lite replications
+* REST API endpoints
+* Couchbase Server
+
+Additionally, the REST API will require authentication using Couchbase Server RBAC user account credentials for both the Admin and Metrics API -- see {rest-api-access--xref}.
+
+We strongly recommend always having TLS enabled, although this can be over-ridden for *development and testing only*.
+
+
+// [#lbl-rest-api-auth]
+// == REST API Authentication
+
+// === Admin and Metrics API
+
+// Administrators typically use the Admin REST API to manage {sgw}.
+
+// From 3.0 they will need to use Couchbase Server RBAC users as basic authentication credentials; not {sgw} users.
+// This applies also to users of the Metrics REST API.
+
+// You can find more about this in {rest-api-access--xref}
+
+// === Public REST API
+
+// Users of the public REST API can continue to authenticate using Sync GAteway users and
+// The role and the user assigned to it can be created using the ADMIN API endpoints.
+// Then, the Sync Function's {sync-function-api-require-role-cmd--xref} helper function can be used to allow certain operations only if the user has that role.
+
+== Configuration
+
+By default {sgw-te} requires TLS encryption.
+The content in <<tbl-tls-config-options>> shows the configuration options used to control this.
+The options include flags that will allow you to override the requirement to use TLS in testing and-or development environments *only*.
+
+.Configuration options controlling TLS-encryption
+[#tbl-tls-config-options,cols="4m,7m,^1a,10a", options="header"]
+|===
+
+|Bootstrap Configuration
+|Command-line flag
+|Default
+|Purpose
+
+|bootstrap.use_tls_server
+|-bootstrap.use_tls_server
+|`true`
+|Default: TLS enabled +
+Set this `false` to allow use of non-secure protocols (`couchbase://` instead of `couchbases://`) -- *development and testing only*
+
+|bootstrap.server_tls_skip_verify
+|-bootstrap.server_tls_skip_verify
+|`false`
+|Default: TLS enabled +
+Set this `true` to skip validation of any certificates received from the server with the CA certificate -- *development and testing only*
+
+|api.https.tls_key_path
+|-api.https.tls_key_path
+|nil
+|Provide the path footnote:[pass:q,a[This can be absolute, or relative to the {sgw} executable's directory]] to the TLS private key file.
+
+Set to `""` or _nil_ to use _plaintext_ -- development or test use *only*
```

Comment:
Cannot be set to `nil` or else Sync Gateway fails to start

Reply:
changed 
Done -- https://github.com/couchbase/docs-sync-gateway/pull/541